packages/cms-lib/__tests__/hubdb.js

```diff
@@ -0,0 +1,55 @@
+const fs = require('fs-extra');
+const { downloadHubDbTable, createHubDbTable } = require('../hubdb');
+const { publishTable } = require('../api/hubdb');
+const { getCwd } = require('../path');
+const hubdbJson = require('./fixtures/hubdb/hubdbTableData');
+
+jest.mock('../path');
+jest.mock('../api/hubdb');
+
+describe('cms-lib/hubdb', () => {
+  it('downloads hubdb table', async () => {
+    const portalId = 123;
+    const tableId = 456;
+    const destPath = 'tmp.json';
+    const projectCwd = '/home/tom/projects';
+
+    getCwd.mockReturnValue(projectCwd);
+
+    const { filePath } = await downloadHubDbTable(portalId, tableId, destPath);
+
+    describe('transforms column names to ids', () => {
+      expect(fs.outputFile).toHaveBeenCalledWith(
+        `${projectCwd}/${destPath}`,
+        expect.stringContaining('"name": "My Event",')
```

Comment:
Minor thing, but it might be work using `JSON.parse()` here and then inspecting the object instead of inspecting a string. The benefit is that then we can verify that the shape of the object matches the expected.

Reply:
Ah, I'm glad that you mentioned that. I updated this, I think that's a lot more clear now.